defi/src/adaptors/db-utils/adaptor-record.ts

```diff
@@ -176,23 +177,37 @@ export const storeAdaptorRecord = async (adaptorRecord: AdaptorRecord, eventTime
         }, (currentData ?? {}) as IRecordAdaptorRecordData),
         eventTimestamp
     }
-    const obj2StoreRemoveReserveKeys = Object.entries(obj2Store).reduce((acc, [key, value]) => {
-        if (dynamoReservedKeywords.includes(key.toUpperCase())) {
-            delete acc[key];
-            return acc
-        }
-        acc[key] = value
-        return acc
-    }, {} as IRecordAdaptorRecordData)
-
+    // const obj2StoreRemoveReserveKeys = Object.entries(obj2Store).reduce((acc, [key, value]) => {
+    //     if (dynamoReservedKeywords.includes(key.toUpperCase())) {
+    //         delete acc[key];
+    //         return acc
+    //     }
+    //     acc[key] = value
+    //     return acc
+    // }, {} as IRecordAdaptorRecordData)
+    adaptorRecord.data = obj2Store 
+    const record = adaptorRecord.getCleanAdaptorRecord()
+    const { PK, SK } = adaptorRecord.keys()
 
     try {
-        console.log("Storing", obj2StoreRemoveReserveKeys, adaptorRecord.keys())
-        await dynamodb.update({
-            Key: adaptorRecord.keys(),
-            UpdateExpression: createUpdateExpressionFromObj(obj2StoreRemoveReserveKeys),
-            ExpressionAttributeValues: createExpressionAttributeValuesFromObj(obj2StoreRemoveReserveKeys)
-        }) // Upsert like
+        console.log("Storing", adaptorRecord.keys())
+        if (record) {
+            await dynamodb.put(record.toItem())
+        }
+        if (errorData) {
+            Object.entries(errorData).forEach(([chain, error]) => {
+                if (dynamoReservedKeywords.includes(chain.toUpperCase())) {
+                    errorData[replaceReservedKeyword(chain)] = error
+                    delete errorData[chain];
+                }
+            })
+            await dynamodb.put({ PK: `${PK}#error`, SK, ...errorData })
```

Comment:
i would not store errors in ddb, instead lets log them somewhere and thats it, at most could store them into some ancilliary db for debugging like we did for tvl, but since that ended up not even being used its probably not worth it